File: Beecrowd/C/2152 Pepe, I Already Took the Candle!.c
#include <stdio.h>

int main()
{
    int i,t, H,M,O;

    scanf("%d",&t);
    for(i=0;i<t;i++){
        scanf("%d %d %d",&H,&M,&O);
        if (H <10){
           printf("0%d:", H);
        }
        else printf("%d:", H);
    
        if(M <10){
        printf("0%d", M);
        }
        else printf("%d", M);
        
    if (O<1)
        printf(" - A porta fechou!\n");
    else
        printf(" - A porta abriu!\n");
        
    }

    return 0;
}

File: Beecrowd/Python/2152 Pepe, I Already Took the Candle!.py
for i in range(int(input())):
    H,M,O =map(int,input().split())
    if H <10:
        H='0'+str(H)

    if M <10:
        M='0'+str(M)
    
    if O<1:
        print(f"{H}:{M} - A porta fechou!")
    else:
        print(f"{H}:{M} - A porta abriu!")
